src/main/webapp/app/exercises/shared/example-submission/example-submissions.component.ts

```diff
@@ -39,4 +44,12 @@ export class ExampleSubmissionsComponent implements OnInit {
             },
         );
     }
+
+    /**
+     * Navigates to the detail view of the example submission
+     * @param id id of the submission or new for a new submission
+     */
+    navigateToExampleSubmission(id: number | 'new') {
+        this.router.navigate([id], { relativeTo: this.activatedRoute });
```

Comment:
We moved away from relative routes to explicit ones to reduce the amount of errors when routes change, so this feels like a step back.

Reply:
Thats interesting, when I found out this is possible it felt like a progress since now this code does not need any changes even if the route changes, as long as the relation between overview and details stays the same, whereas explicitly writing down the url for non-exam and exam exercises is just a duplication that makes route changes more difficult. The code line with the route can be searched for more easily that way, but that seems like a minor advantage. But if this is the guideline I will change it.